dockerfiles/testsuite/start_test.sh

```diff
@@ -121,6 +117,8 @@ e2etest_cenususizelection () {
 
 log "### Starting test suite ###"
 $COMPOSE_CMD build
+$COMPOSE_CMD up -d seed # start the seed first so the nodes can properly bootstrap
+sleep 10
```

Comment:
10 sec seems too much, I think 5 sec is actually enough

Reply:
Its not. The seed needs to be able to bootstrap the network, in a slow environment such as some CIs, it might require around 10 seconds.